lib/boxr/files.rb

```diff
@@ -128,8 +128,10 @@ def upload_file(path_to_file, parent, name: nil, content_created_at: nil, conten
       file_info.entries[0]
     end
 
-    def upload_new_version_of_file(path_to_file, file, content_modified_at: nil, send_content_md5: true,
+    def upload_new_version_of_file(path_to_file, file, name: nil, content_modified_at: nil, send_content_md5: true,
```

Comment:
you should put new parameters, even those with a name, at the end of the list so as not to break peoples code if they call this method without specifying the parameter names (positional only).